app/src/main/java/de/tum/in/tumcampusapp/api/TUMCabeClient.java

```diff
@@ -56,6 +57,8 @@
     private static final String API_OWN_QUESTIONS = "question/my/";
     private static final String API_FACULTY = "faculty/";
 
+    private static final String API_WIFI_HEATMAP = "wifimap/";
+
```

Comment:
Please fix the blank lines here. Move the thing one up and remove that added blank line